src/index.js

```diff
@@ -3,6 +3,7 @@ require('angularjs-dragula/dist/angularjs-dragula.js');
 require('jmap-draft-client/dist/jmap-draft-client.js');
 require('ui-select/dist/select.js');
 require('esn-frontend-common-libs/src/frontend/vendor-libs.js');
+require('esn.inbox.libs/app/app.module.js');
```

Comment:
Should not require it here but in the `src/linagora.esn.unifiedinbox/app/app.js`